apps/examples/wifi_manager/wm_test/wm_test_connectbyrssi_test.c

```diff
@@ -0,0 +1,311 @@
+/****************************************************************************
+ *
+ * Copyright 2022 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License\n");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+#include <tinyara/config.h>
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+#include <stdint.h>
+#include <unistd.h>
+#include <sys/types.h>
+#include <semaphore.h>
+#include <errno.h>
+#include <wifi_manager/wifi_manager.h>
+#include <netutils/netlib.h>
+#include "wm_test.h"
+#include "wm_test_log.h"
+
+static sem_t g_wio_sem;
+static struct wt_options *opt;
+static char bssid[20];
+static char connected_ap_mac[20];
+
+#define TAG "[WTCR]"
+
+#define WIO_TEST_SIGNAL								\
+	do {											\
+		sem_post(&g_wio_sem);			\
+		WT_LOG(TAG, "send signal");	\
+	} while (0)
+
+#define WIO_TEST_WAIT								\
+	do {											\
+		WT_LOG(TAG, "wait signal");	\
+		sem_wait(&g_wio_sem);				\
+	} while (0)
+
+
```

Comment:
Could you please avoid this extra line space?

Reply:
Done